code/modules/reagents/reagent_containers/food/drinks/bottle.dm

```diff
@@ -278,6 +278,15 @@
 		..()
 		reagents.add_reagent("gin", 100)
 
+/obj/item/weapon/reagent_containers/food/drinks/bottle/gin/empty_gin
```

Comment:
For empty bottles, please use the nomenclature `bottle/type/empty`. Use `reagents.del_reagents() ` to properly remove all reagents from the bottle.